senza/components/elastic_load_balancer.py

```diff
@@ -116,7 +116,9 @@ def component_elastic_load_balancer(definition, configuration, args, info, force
                                               health_check_port,
                                               health_check_path)
 
-    if configuration.get('NameSuffix'):
+    if configuration.get('LoadBalancerName'):
+        loadbalancer_name = info["LoadBalancerName"]
```

Comment:
If you are trying to resolve a load balancer name like "example-app-integration-{{SenzaInfo.StackVersion}}", then would you not rather do this? 
```
loadbalancer_name = get_load_balancer_name(info["LoadBalancerName"], info["StackVersion"])
```
Where you set LoadBalancerName to "example-app-integration".